src/index.html

```diff
@@ -1,17 +1,157 @@
-<!doctype html>
+
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>The form</title>
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      method="POST"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="inputs-block">
+        <legend>Personal information</legend>
+
+        <label class="input-row">
+          Surname:
```

Comment:
This way of wrapping with label is kinda unorthodox, I'd suggest in the future do it the "regular way" without wrapping and using 'for' and 'id' attributes for the sake of readability,

Reply:
For now just add an empty line between all label titles and inputs e.g.: 
```html
<label class="input-row">
  Surname:

  <input name="surname" autocomplete="off">
</label>
```
Do it for all your labels with inputs